extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -115,6 +116,12 @@ export class WorkspaceService implements IWorkspaceService {
 				currentProjects.push(projectFile);
 				newProjectFileAdded = true;
 
+				TelemetryReporter.createActionEvent(TelemetryViews.WorkspaceTreePane, 'add project to workspace')
```

Comment:
why do some action events have spaces in the action name and others don't?

Reply:
Because I don't know what the preferred convention is. 😄